pkg/blockchaininfo/nats_publisher.go

```diff
@@ -0,0 +1,235 @@
+package blockchaininfo
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/nats-io/nats-server/v2/server"
+	"github.com/nats-io/nats.go"
+	"github.com/wavesplatform/gowaves/pkg/proto"
+)
+
+const StoreBlocksLimit = 200
+const PortDefault = 4222
+const HostDefault = "127.0.0.1"
+const ConnectionsTimeoutDefault = 5 * server.AUTH_TIMEOUT
+
+const NatsMaxPayloadSize int32 = 1024 * 1024 // 1 MB
+
+const PublisherWaitingTime = 100 * time.Millisecond
+
+const (
+	StartPaging = iota
+	EndPaging
+	NoPaging
+)
+
+type BUpdatesExtensionState struct {
+	currentState  *BUpdatesInfo
+	previousState *BUpdatesInfo // this information is what was just published
```

Comment:
Maybe store info in a struct with `BlockUpdatesInfo` and transformed map of data entries (key -> marshaled value)?